lib/ReactViews/Notification/MapInteractionWindow.tsx

```diff
@@ -77,6 +79,10 @@ class MapInteractionWindow extends Component<{
     );
   }
 
+  componentDidUpdate(): void {
+    this.observableViewState = this.props.viewState;
```

Comment:
This should be wrapped in action if we finalize on using observable view state.

![image](https://github.com/user-attachments/assets/9ec7bbbc-191b-4c36-a9bc-db8ce6a95284)

Reply:
fixed